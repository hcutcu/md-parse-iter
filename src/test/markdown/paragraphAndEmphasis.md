A new paragraph shows emphasis on the text. now this is **bold** and this is *italic* and this is ***bold italic*** and this is ***`code-inline bold italic`***.  
and also have a new line with a break.