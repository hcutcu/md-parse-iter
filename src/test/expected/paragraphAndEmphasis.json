[
  {
    "type": "paragraph",
    "children": [
      {
        "type": "span",
        "content": "A new paragraph shows emphasis on the text. now this is ",
        "attributes": []
      },
      {
        "type": "span",
        "content": "bold",
        "attributes": ["strong"]
      },
      {
        "type": "span",
        "content": " and this is ",
        "attributes": []
      },
      {
        "type": "span",
        "content": "italic",
        "attributes": ["em"]
      },
      {
        "type": "span",
        "content": " and this is ",
        "attributes": []
      },
      {
        "type": "span",
        "content": "bold italic",
        "attributes": ["em", "strong"]
      },
      {
        "type": "span",
        "content": " and this is ",
        "attributes": []
      },
      {
        "type": "span",
        "content": "code-inline bold italic",
        "attributes": ["code-inline", "em", "strong"]
      },
      {
        "type": "span",
        "content": ".",
        "attributes": []
      },
      {
        "type": "lineBreak"
      },
      {
        "type": "span",
        "content": "and also have a new line with a break.",
        "attributes": []
      }
    ]
  }
]
